### PR TITLE
New design sahar

### PR DESCRIPTION
**Pull request by SaharMehrpour**
I implemented some pages. The style needs to be improved a lot! The primary goal of this PR is to discuss the pages and their contents. I tried to follow David's guidelines as much as possible. I removed the `About` page as I assumed it is redundant (?). We can add a footer with all the links for GitHub and the Lab location, etc.

# Home page
It presents 3 recent news and all current projects. (Right now all our projects are active)
home

# Projects page
It lists all projects (current and archived ones). For more info, click on `More...` 
projects

# [individual] Project page
It includes all information about the project. Any long description with images and videos (in the form of `md`), collaborators, and publications.
project

# People page
people

# Member page
It displays all information about the member, bio, publications, projects, etc.
member

# Gallery
Gallery

### CODE REVIEW

**Comment by SaharMehrpour** [comment]
Demo: https://saharmehrpour.github.io/devx.website/
Code: https://github.com/SaharMehrpour/devx.website

**Comment by Alaboudi1** [comment]
The design looks good! Also, thanks for the readme. My question is, it seems most of the work is done inside doc_src, but why we still have a lot of files inside doc? For example, we have docs_src/_layouts/member.html and docs/people/index.html . These files are duplicate, right?

**Comment by SaharMehrpour** [comment]
`docs` files are auto-generated from `docs_src`. Basically, we can ignore the `docs` folder and its contents.

David wrote a [script](https://github.com/devuxd/devx.cs.gmu.edu/blob/master/.github/workflows/deploy_docs.yml) that builds the website (`docs` folder which is the source for GitHub pages) automatically (after each commit to the master I think). 

David explained in Slack that in normal Jekyll projects, there is no need in having this script and generally having a `docs` folder. But here we are using some libraries that require custom build scripts.

**Comment by Alaboudi1** [comment]
I see, thanks for the  explanation. I guess then if that generated automatically, we do not need to keep it in master. The script can generate the file and push it to another bransh (e.g., gh-pages and make GitHub depoly that bransh). This will master less clutter. I guess @luminaxster needs to tell us if that is a possible solution.

File: docs_src/_includes/footer.html [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<footer>
+    Copy Right
```
Comment (luminaxster) [inline docs_src/_includes/footer.html:2]: What do we want here? the default? or year?

File: docs_src/_pagedata/navbarItems.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+---
+for: header
+items: [home,projects,publications,people,gallery]
```
Comment (luminaxster) [inline docs_src/_pagedata/navbarItems.md:3]: do we need a home link? Maybe the logo is enough.

**Review by luminaxster — APPROVED** [review]

**Pull request merged**